conf/chunkserver.conf.example

```diff
@@ -92,6 +92,16 @@ copyset.check_retrytimes=3
 copyset.finishload_margin=2000
 # 循环判定copyset是否加载完成的内部睡眠时间
 copyset.check_loadmargin_interval_ms=1000
+# scan copyset interval
+copyset.scan_interval_sec=5
+# the size each scan 4MB
```

Comment:
why change this default value to 4MB?

Reply:
> why change this default value to 4MB?

It will take a long time if scansize is 128KB, there are similar time cost between 128KB and 4MB after test.